examples/storage/couchdb/images/init_couch/configer_couch.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+fqdn=$(hostname --fqdn)
+
+cp /core-config/default.ini $INSTALL_LOCATION/etc/default.ini
+cp /core-config/local.ini $INSTALL_LOCATION/etc/local.ini
+cp /core-config/vm.args $INSTALL_LOCATION/etc/vm.args
+mkdir $INSTALL_LOCATION/etc/local.d
+mkdir $INSTALL_LOCATION/etc/default.d
+sed -i $INSTALL_LOCATION/etc/vm.args -e "s/^\-name db/\-name db@$fqdn/"
```

Comment:
This last one puzzles me; Erlang should default to using the FQDN when the node boots up. Is this shell script evaluated at a different time? I don't know why `hostname --fqdn` would work here but Erlang would fail to boot with `-name db`